### PR TITLE
Fix revert. Ковальчук Александр. Задача 1. Вариант 13. Максимальное значение элементов матрицы

### PR DESCRIPTION
**Pull request by KADCHDR**
https://github.com/learning-process/ppc-2024-autumn/pull/181  performance has been improved by improving the algorithm and eliminating unnecessary checks.

### CODE REVIEW

File: tasks/seq/kovalchuk_a_max_of_vector_elements/src/ops_seq.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+bool kovalchuk_a_max_of_vector_elements_seq::TestSequentialTask::run() {
+  internal_order_test();
+  if (!input_.empty()) {
+    std::vector<int> local_res(input_.size());
+    for (unsigned int i = 0; i < input_.size(); i++) {
+      if (!input_[i].empty()) {
+        local_res[i] = *std::max_element(input_[i].begin(), input_[i].end());
+      } else {
+        local_res[i] = INT_MIN;
+      }
```
Comment (allnes) [inline tasks/seq/kovalchuk_a_max_of_vector_elements/src/ops_seq.cpp:42]: please check it in validation and not process in run()

File: tasks/seq/kovalchuk_a_max_of_vector_elements/func_tests/main.cpp [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+#include <random>
+#include <vector>
+
+#include "seq/kovalchuk_a_max_of_vector_elements/include/ops_seq.hpp"
+
+using namespace kovalchuk_a_max_of_vector_elements_seq;
+
+std::vector<int> getRandomVector(int sz, int min = MINIMALGEN, int max = MAXIMUMGEN);
+std::vector<std::vector<int>> getRandomMatrix(int rows, int columns, int min = MINIMALGEN, int max = MAXIMUMGEN);
+
```
Comment (aobolensk) [inline tasks/seq/kovalchuk_a_max_of_vector_elements/func_tests/main.cpp:13]: Please either wrap with namespace or make static to avoid collisions

**Review by aobolensk — CHANGES_REQUESTED** [review]

**Comment by aobolensk** [comment]
Please, note what was the fix in the description

File: tasks/seq/kovalchuk_a_max_of_vector_elements/func_tests/main.cpp (hunk 2 above)
Comment (KADCHDR) [reply tasks/seq/kovalchuk_a_max_of_vector_elements/func_tests/main.cpp]: fixed

File: tasks/seq/kovalchuk_a_max_of_vector_elements/src/ops_seq.cpp (hunk 1 above)
Comment (KADCHDR) [reply tasks/seq/kovalchuk_a_max_of_vector_elements/src/ops_seq.cpp]: fixed

**Pull request merged**